hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -349,6 +350,7 @@ public ResponseCodeEnum validate(final TransactionBody contractCreateTxn) {
 
     Bytes prepareCodeWithConstructorArguments(final ContractCreateTransactionBody op) {
         if (op.getInitcodeSourceCase() == INITCODE) {
+            validateFalse(op.getInitcode().isEmpty(), CONTRACT_BYTECODE_EMPTY);
```

Comment:
Can't decide whether it's a good idea or not a good idea that the error returned here isn't the same as the error returned at L366.

Reply:
I think it's fine as this scenario is for the non file case right?